apps/api/src/routes/api/admin/foods/global/index.ts

```diff
@@ -15,5 +15,10 @@ export default () => {
     .get(wrapAsync(adminGlobalFoodsController.read))
     .put(wrapAsync(adminGlobalFoodsController.update));
 
+  // TODO: V4-1016 DELETE ONCE DONE
+  // router
+  //   .route('/rebuild')
```

Comment:
I would move the API endpoints to `api/admin/locales`, like POST `api/admin/locales/rebuild` as it's locales focused?

Reply:
This "rebuild all routes" endpoint doesn't make sense anymore since there is no scenario where the admin can trigger a rebuild of all locales. It is still possible via the Specific Locale Rebuild if something will put "all" into the Redis Cache. So we can use this if we would like to give the ability to trigger the rebuild for all locales via the admin dashboard in the future.